walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -4,68 +4,105 @@
     <meta charset="UTF-8">
     <title>All About Jalyn 🐈</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
-<div class="navigation_bar">
-    
+    	<div id="navigation-bar">
+        	<span class="nav nav-btn"> <a class="nav" href= "index.html" style="background-color: white">Home</a> </span>
+        	<span class="nav nav-btn"> <a class="nav" href="Resume_CS.pdf" style="background-color: white">Resume</a> </span>
+        	<span class="nav nav-btn"> <a class="nav" href="/socials.html" style="background-color: white">Socials</a> </span>
+    	</div>
 
-	<h3><a class="nav" href="/random.html">Link One</a></h3>
+        <div class="full-page">
```

Comment:
combine full-page div and first-page div into one?

Reply:
The CSS for full page apply to multiple divs while the first page CSS only applies to that one div tag. I changed it from a class to an id to clarify